Jenkinsfile

```
@@ -45,7 +45,7 @@ pipeline {
     agent { label 'lightweight' }
 
     triggers {
-        cron(env.BRANCH_NAME == 'master' ? 'TZ=America/Toronto\n0 0 * * *\n' : '' +
+        cron(env.BRANCH_NAME == 'master' ? 'TZ=GMT\n0 1 * * *\n' : '' +
```

My math says that 4 hours earlier, at least currently, during DST[1], is midnight UTC/GMT.  Probably also more correct to use UTC rather than GMT:

```suggestion
        cron(env.BRANCH_NAME == 'master' ? 'TZ=UTC\n0 0 * * *\n' : '' +
```

[1] But this problem of DST and when the test will start is why I didn't choose UTC/GMT.  I chose a timezone that would move with DST so that the start time was always relevant/consistent with most people's clocks.

---

I was just going with an actual timezone as UCT is technically not a time zone but a time standard.  That said both exist in `/usr/share/zoneinfo` so either should work.  London currently +5 hours of EDT that's why I used 1 am.